### PR TITLE
refactor baseline pubsub test out of private repo

### PR DESCRIPTION
**Pull request by yusefnapora**
This pulls in the baseline evaluation plan from the private repo we've been using to evaluate gossipsub under attack. 

We're planning to publish that code soon also, but for now this just has the "honest" node type included.

cc @daviddias  @vyzo @dirkmc @nonsense 

BTW, if anyone wants to give me write access to this repo, that would be sweet 😄

### CODE REVIEW

**Comment by jacobheun** [comment]
@yusefnapora access should be fixed, I added the `Repos - Go` team.

File: pubsub/README.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+Because the full tracer output is quite large (several GB for a few minutes of test execution with lots of nodes),
+we aggregate the trace events at runtime and spit out a json file with aggregated metrics at the end of the test.
+We also capture a filtered subset of the original traces, containing only Publish, Deliver, Graft, and Prune events.
+At the end of the test, we run [tracestat](https://github.com/libp2p/go-libp2p-pubsub-tracer/blob/master/cmd/tracestat/main.go)
+on the filtered traces to calculate the latency distribution and get a summary of publish and deliver counts.
+
+### Note about "attackers"
+
+There are some references in the code to "attacker" peers and "sybils". The attack code has been omitted from the
+initial public release, however, we plan to re-introduce it in the near future.
```
Comment (daviddias) [inline pubsub/README.md:322]: I recommend to just fully omitting this. We want this to be the baseline test only and avoid overloading users with information.

File: pubsub/README.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+to another location by setting the `TESTGROUND_HOME` environment variable.
+
+#### Cloning this repo
+
+The testground client will look for test plans in `$TESTGROUND_HOME/plans`, so this repo should be cloned or
+symlinked into there:
+
+```shell
+cd ~/testground/plans # if this dir doesn't exist, run 'testground list' once first to create it
+git clone git@github.com:libp2p/test-plans libp2p-plans
```
Comment (daviddias) [inline pubsub/README.md:64]:
```suggestion
git clone git@github.com:libp2p/test-plans
```

**Comment by yusefnapora** [comment]
I've tested this out on our jupyterhub instance using the k8s runner. Everything seems to work as expected, but it was taking a fairly long time to get all pods in the Ready state - about 2.5 minutes for 1k nodes.

I increased the setup time to 3 minutes, since if it's lower than the actual time needed to get into the running state, the test results get skewed.

**Pull request merged**